.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Please include a summary of the change and which issue is fixed or which feature is added.
+
+- [ ] Issue 1 fixed
+- [ ] Issue 2 fixed
+- [ ] Feature 1 added
+- [ ] Feature 2 added
```

Comment:
Can we here mention the commit message format (while squashing) should be:

```
One line summary of the PR

* Bullet points
  - sub point 1
  - sub point 1
* Another bullet point
  - sub point 1

fixes #issueId

Co-authored-by : Name <email>
```

Reply:
That is more for integrator role. However we cannot add a template for `Squash and merge`

Reply:
I think that this is more for the contributing guideline and not the PR description template. This is something that needs to be communicated and agreed with everyone who has permission for merging in the repo I believe.

Reply:
sound reasonable!